cidemipyme-gatsby/src/components/sections/ClientsSection.jsx

```diff
@@ -0,0 +1,133 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  Col,
+  Container,
+  Row,
+} from 'reactstrap';
+import Slider from "react-slick";
+
+import client_1 from 'images/client_1.png';
```

Comment:
you can try to do this dynamically, but I am not sure if it will work.

```javascript
const clientLogos = Array.from({length: 30}, (nil, index) => require(`client_${index + 1}`));
```